### PR TITLE
feat(xdsl.meetings): add endpoints to retrieve and select meeting

### PR DESCRIPTION
**Pull request by Steffy29**
Add endpoints to retrieve available meetings and to order one for xDSL customer.

Ref: UXCT-194

### CODE REVIEW

File: src/api/xdsl/meeting/xdsl-meeting.v6.service.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+angular.module('ovh-api-services').service('OvhApiXdslMeetingV6', ($resource, OvhApiXdslMeeting, Poller) => {
+  const interceptor = {
+    response(response) {
+      OvhApiXdslMeeting.resetCache();
+      return response.resource;
+    },
+  };
+
+  const meeting = $resource('/xdsl/:serviceName/orderMeeting', {
```
Comment (antleblanc) [inline src/api/xdsl/meeting/xdsl-meeting.v6.service.js:9]: Regarding the `url`, both `orderMeeting` and `searchOrderMeetings` actions must be part of the following service `OvhApiXdslV6`.

Is that make sense for you?

Comment (Steffy29) [reply src/api/xdsl/meeting/xdsl-meeting.v6.service.js:9]: If you want, it's the same for me.

**Comment by ovh-cds** [comment]
CDS Report build#1603.0 ✘  
* 
  * build ✘  
  * test ✘

File: src/api/xdsl/meeting/xdsl-meeting.v6.service.js (hunk 1 above)
Comment (antleblanc) [reply src/api/xdsl/meeting/xdsl-meeting.v6.service.js]: This is how the project is architecture.
You can refer to the following documentation here: https://github.com/ovh-ux/ovh-api-services#structure

Thank you!

**Comment by ovh-cds** [comment]
CDS Report build#1604.0 ✘  
* 
  * build ✘  
  * test ✘

**Pull request merged**